qutip/solver/ode/scipy_integrator.py

```diff
@@ -277,17 +304,16 @@ def _one_step(self, t):
         # Here we want to advance up to t doing maximum one step.
         # We check if a new step is needed.
         t_front = self._ode_solver._integrator.rwork[12]
+        safe_delta = self._ode_solver._integrator.rwork[11]/10 + 1e-15
```

Comment:
Could you explain a little what the safe_delta change does and why it is better? I'm happy for it to stay in this PR but it seems unrelated to the concurrency checks?